libs/deal-ticket/src/components/info-market.tsx

```diff
@@ -8,17 +8,129 @@ import {
 import {
   KeyValueTable,
   KeyValueTableRow,
+  AsyncRenderer,
+  Splash,
   Accordion,
 } from '@vegaprotocol/ui-toolkit';
 import startCase from 'lodash/startCase';
 import pick from 'lodash/pick';
 import omit from 'lodash/omit';
-import type { DealTicketQuery_market } from './__generated__/DealTicketQuery';
+import type {
+  MarketInfoQuery,
+  MarketInfoQuery_market,
+} from './__generated__/MarketInfoQuery';
 import BigNumber from 'bignumber.js';
+import { gql, useQuery } from '@apollo/client';
+
+const MARKET_INFO_QUERY = gql`
+  query MarketInfoQuery($marketId: ID!) {
```

Comment:
chore (blocking): Can we add a generateMarketInfoQuery mock, and add it to our tests. Otherwise we'll get flakey tests

Reply:
added.